webapp/src/plugin.jsx

```diff
@@ -24,6 +30,16 @@ import {
 } from './actions';
 import reducer from './reducer';
 
+function getLanguageTest(locale) {
```

Comment:
why is function called getLanguage _Test_?

Reply:
Because I forgot to change it.